packages/schematics/angular/application/index.ts

```diff
@@ -264,13 +260,20 @@ export default function (options: ApplicationOptions): Rule {
     const workspace = await getWorkspace(host);
     const newProjectRoot = (workspace.extensions.newProjectRoot as string | undefined) || '';
     const isRootApp = options.projectRoot !== undefined;
+
+    // If scoped project (i.e. "@foo/bar"), convert dir to "foo/bar".
+    let folderName = options.name.startsWith('@') ? options.name.substr(1) : options.name;
+    if (/[A-Z]/.test(folderName)) {
```

Comment:
Can you add test cases for the scoped and un-scoped dasherized cases? (`@foo/BarBaz`, etc.)

Reply:
Added some more tests.